engine/src/main/java/org/hibernate/search/util/logging/impl/Log.java

```diff
@@ -736,4 +737,6 @@
 	@Message(id = 248, value = "WorkList should never be empty. Stacktrace below \n %s" )
 	void workListShouldNeverBeEmpty(String stackTrace);
 
+	@Message(id = 249, value = "Unsupported value type for configuration property " + Environment.ERROR_HANDLER + ": %1$s")
+	SearchException unsupportedErrorHandlerConfigurationValueType(@FormatWith(ClassObjectFormatter.class) Class<?> errorHandlerValueType);
```

Comment:
> Since you are pointing these cases out so nicely. What's about fixing
them as well using @FormatWith? I really dislike the printing of class.

I knew that would be coming over me ;)

Yes, I can do that.